bokeh/models/renderers.py

```diff
@@ -242,7 +242,7 @@ def __init__(self, **kwargs) -> None:
     being hovered over by a ``HoverTool``.
     """)
 
-    muted_glyph = Nullable(Instance(Glyph), help="""
+    muted_glyph = Either(Auto, Instance(Glyph), Null, default="auto", help="""
```

Comment:
This should still be possible
```suggestion
    muted_glyph = Nullable(Either(Auto, Instance(Glyph), default="auto", help="""
```
We may explicitly utilize the presence of `Nullable` in the docs generation in the future, so this form is preferred.

Reply:
@bryevdv thanks. I wrote it that way to keep it consistent with `selection_glyph` and `non_selection_glyph` (see below). Thoughts on updating those to `Nullable(Either())` while I'm here?

https://github.com/bokeh/bokeh/blob/d771efcdb195c9b9e27e63a4012cb58d87f7be14/bokeh/models/renderers.py#L224-L238

Reply:
> Thoughts on updating those to Nullable(Either()) while I'm here?


👍  Yes, that would be great!

Reply:
Changed to Nullable(Either()) in 0b1c81f